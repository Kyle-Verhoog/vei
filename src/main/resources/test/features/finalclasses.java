  public final class Finalclasses {
    public Finalclasses() {}
  }