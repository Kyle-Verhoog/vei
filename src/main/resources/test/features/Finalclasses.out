PUBLIC ┃ public
FINAL ┃ final
CLASS ┃ class
IDENTIFIER ┃ A
{ ┃ {
PUBLIC ┃ public
IDENTIFIER ┃ A
( ┃ (
) ┃ )
{ ┃ {
} ┃ }
} ┃ }